artiq/firmware/libboard_artiq/hmc830_7043.rs

```diff
@@ -113,18 +80,66 @@ mod hmc830 {
     }
 
     pub fn init() -> Result<(), &'static str> {
```

Comment:
This doesn't need a `Result<>` anymore.

Reply:
thanks! will fix.

Do you want this all squashed into a single commit at the end?